groups/alteryx-sandbox/security_groups.tf

```diff
@@ -72,6 +83,15 @@ resource "aws_vpc_security_group_ingress_rule" "ec2_http" {
   to_port                      = 80
 }
 
+resource "aws_vpc_security_group_ingress_rule" "ec2_worker_api" {
+  description       = "Worker api access"
+  security_group_id = module.alteryx_worker_ec2_security_group.this_security_group_id
+  prefix_list_id    = data.aws_ec2_managed_prefix_list.ci.id
+  ip_protocol       = "tcp"
+  from_port         = 80
+  to_port           = 80
+}
```

Comment:
The rule config should be reviewed / verified as it doesn't provide the access described in the PR.